<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>597</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Iron Road</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTabWidget" name="mainWindowWidget">
      <property name="focusPolicy">
       <enum>Qt::TabFocus</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="bd">
       <attribute name="title">
        <string>БД</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTableView" name="mainViewDB"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="admin">
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <attribute name="title">
        <string>Администратирование</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="selectAct">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>Добавить запись</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Удалить запись</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Изменить запись</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Поиск </string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedActionsWidget">
          <property name="tabletTracking">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="midLineWidth">
           <number>0</number>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="page_Add">
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="1">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="trainIDAdd">
              <property name="text">
               <string>ID Поезда</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="trainIDAddEdit"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="destination">
              <property name="text">
               <string>Пункт назначения</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="destinationEdit"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="departureTime">
              <property name="text">
               <string>Время отправления</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDateTimeEdit" name="departureTimeEdit"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="JorneyTime">
              <property name="text">
               <string>Время в пути</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QTimeEdit" name="JorneyTimeEdit"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="AvaiableSeats">
              <property name="text">
               <string>Свободные места</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="AvaiableSeatsEdit"/>
            </item>
            <item row="7" column="1">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="6" column="1">
             <widget class="QPushButton" name="buttonAdd">
              <property name="text">
               <string>Добавить</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_Delete">
           <layout class="QFormLayout" name="formLayout">
            <item row="1" column="0">
             <widget class="QLabel" name="trainIDelete">
              <property name="text">
               <string>ID Поезда</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="trainIDeleteEdit"/>
            </item>
            <item row="3" column="1">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1">
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="buttonDelete">
              <property name="text">
               <string>Удалить</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_Update">
           <layout class="QFormLayout" name="formLayout_3">
            <item row="0" column="1">
             <spacer name="verticalSpacer_5">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="trainIDUpdate">
              <property name="text">
               <string>ID Поезда</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="trainIDUpdateEdit"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="destination_2">
              <property name="text">
               <string>Пункт назначения</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="destinationEdit_2"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="departureTime_2">
              <property name="text">
               <string>Время отправления</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="JorneyTime_2">
              <property name="text">
               <string>Время в пути</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QTimeEdit" name="JorneyTimeEdit_2"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="AvaiableSeats_2">
              <property name="text">
               <string>Свободные места</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="AvaiableSeatsEdit_2"/>
            </item>
            <item row="7" column="1">
             <spacer name="verticalSpacer_6">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="6" column="1">
             <widget class="QPushButton" name="buttonUpdate">
              <property name="text">
               <string>Обновить</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDateTimeEdit" name="departureTimeEdit_2">
              <property name="locale">
               <locale language="English" country="UnitedStates"/>
              </property>
              <property name="accelerated">
               <bool>false</bool>
              </property>
              <property name="showGroupSeparator" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_Search">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="search">
              <property name="text">
               <string>Поиск:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="searchEdit"/>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="buttonSearch">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resource.qrc">
                <normaloff>:/img/search.png</normaloff>:/img/search.png</iconset>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="3">
             <widget class="QTableView" name="resViewSearching"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
